data/_ui/tooltips.txt

```diff
@@ -1092,23 +1092,242 @@ tip "This outfit is considered an atrocity."
 
 
 # Controls
-tip `Reverse`
-	`Turn opposite to the flight direction (or use reverse engines if you have them installed). Combine with <shift> key to stop your ship.`
+tip "Forward thrust"
+	`Accelerate your flagship forward.`
+
+tip "Turn left"
+	`Turn your flagship left.`
+
+tip "Turn right"
+	`Turn your flagship right.`
+
+tip "Reverse"
+	`Turn your flagship opposite its current direction of flight. If reverse thrusters are installed, accelerate backwards. With <shift>, stop your ship.`
+
+tip "Fire afterburner"
+	`Fire your flagship's afterburner, if one is installed.`
+
+tip "Auto steer"
+	`If the "Automatic aiming" setting is off, active automatic aiming when this key is pressed.`
+
+tip "Land on planet / station"
+	`Land on the selected planet. If no planet is selected, automatically selects a planet to land on.`
+
+tip "Initiate hyperspace jump"
+	`Initiate a hyperspace jump to follow your current travel plan. If you have no travel plan, initiates a jump nearest to your current facing angle.`
+
+tip "View star map"
+	`Open the star map panel.`
+
+tip "View player info"
+	`Open the player info panel.`
+
+tip "Select nearest hostile ship"
+	`Select the nearest hostile ship. With ship, select the nearest ship regardless of hostility.`
+
+tip "Select next ship"
+	`Select the next non-escort ship in the system, cycling through all ships. With <shift>, cycle through escorts.`
+
+tip "Talk to selected ship"
+	`Hail the selected ship or planet. If both a ship and planet are selected, hails the ship. With <shift>, hails the planet.`
+
+tip "Board selected ship"
+	`Board the selected ship. If no boardable ship is selected, selects a ship to board based on the "Boarding target priority" setting. With <shift>, selects an escort to board.`
```

Comment:
I don't think it's worth mentioning, but does the same priority setting work for escorts as well?

Reply:
I'll have to look.

Reply:
It does, but it seems redundant to add something like "With <shift>, selects an escort to board according to the aforementioned setting."